src/Http/WebUtilities/src/FileBufferingReadStream.cs

```diff
@@ -467,6 +438,97 @@ async Task CopyToAsyncImpl()
         return CopyToAsyncImpl();
     }
 
+    /// <summary>
+    /// Reads this stream into the buffer.
+    /// </summary>
+    /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
+    public async Task BufferAsync(CancellationToken cancellationToken = default)
```

Comment:
Are we going to try to add this API in rc1?

Reply:
If we get confirmation that it addresses the problem.